todo_app/templates/index.html

```diff
@@ -15,14 +15,32 @@ <h1 class="display-4">To-Do App</h1>
 
   <div class="row">
     <div class="col-md-12">
-      <h2>Items</h2>
+      <h2>Todo Items</h2>
       <ul class="list-group mb-4">
-         {% for item in items %}
+         {% for item in view_model.todo_items %}
          <li>{{ item.name }} -{{ item.status }}</li>
          <form action="complete-item/{{item.id}}" method="post">
             <button>Complete</button>
          {% endfor %}
```

Comment:
I claim you have a bug here - to reproduce and test it out, try:

* Complete all the items in your to-do list, so it's empty
* Add two items, `item1` and `item2`
* Click to mark `item2` as complete
* **Notice that `item1` is the one that got completed!**

The reason for this is because we're not closing off the forms correctly here, which leads the HTML to slightly guess when each form stops, which means it's using the wrong URL/wrong item-id and leading to errors

Instead this should look like:
```jinja2
         {% for item in view_model.todo_items %}
            <li>{{ item.name }} -{{ item.status }}</li>
            <form action="complete-item/{{item.id}}" method="post">
               <button>Complete</button>
            </form>
         {% endfor %}
```
(Note the key penultimate line `</form>`, and also note how a little bit of indenting makes it much easier to see which lines close off a corresponding line, and thus easier to see when we've missed a closing tag!)

Reply:
It also seems to have a side effect of messing with the CSS so that some of your buttons grow huge!